hackforla/VRMS
# Frontend linter 2

This adds code formatting and linter rules. The linter and code formatter are not enforced at this time, but can be used manually if anyone wants to use these tools on their branch. 

Work Done:
I am removing refactor. 
Eslint configured in npm/yarn (not enforced)
Pretter configured in npm/yarn (not enforced)
Prettier configured and disabled on VSCode editor settings.
Prettified and eslint out-of-the-box changes for the admin page only. 

Possible next steps:
Olga to add admin page unit tests. 
Add tests and refactor. 
Review and possibly enforce linters and prettier. 


How to Test:
1. Open the PR locally
2. cd client
3. yarn run eslint FILENAME) --fix
4. yarn prettier --write FILENAME
5. Run in VSCod
4. You will see a number of errors in the console (see screenshot below)
